webextensions/api/scripting.json

```diff
@@ -96,7 +96,8 @@
               "firefox_android": "mirror",
               "opera": "mirror",
               "safari": {
-                "version_added": false
+                "version_added": "16.4",
+                "notes": "Available for use in Manifest V2 or later."
```

Comment:
@jdatapple I'm not sure we need this. I'm just in the process of updating the note on the scripting API to clarify support (given it arrived in Firefox for MV2 earlier than MV3):

> This API is available for use in:
>
> - Manifest V2 in Safari 15.4 and Firefox 102+.
> - Manifest V3 or higher, when supported. For example, it's available in Firefox 109+.